### ISSUE TITLE
Implement `<Listbox>` component

### ISSUE THREAD

**Issue by far-fetched** [GavinJoyce/ember-headlessui#69]
I have already implemented it in one of my pet project. There is a lack of accessibility features there, but at least it would be a good starting point. I just let you know to not repeat work. I can push it forward in spare time.
